<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Samanwoy Mukhopadhyay" />
    <meta name="date" content="2021-04-21" />
    <title>Samanwoy Mukhopadhyay’s resume</title>

    
        <script src="pagedown_cv_files/header-attrs-2.7/header-attrs.js"></script>
        <link href="pagedown_cv_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
        <link href="pagedown_cv_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
        <link href="pagedown_cv_files/paged-0.14/css/resume.css" rel="stylesheet" />
        <script src="pagedown_cv_files/paged-0.14/js/config.js"></script>
        <script src="pagedown_cv_files/paged-0.14/js/paged.js"></script>
        <script src="pagedown_cv_files/paged-0.14/js/hooks.js"></script>
    
    
  </head>
  <body>

<div id="aside" class="section level1">
<h1>Aside</h1>
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto:m.samanwoy@gmail.com" class="email">m.samanwoy@gmail.com</a></li>
<li><i class="fa fa-envelope-square"></i> <a href="mailto:sm3@nibmg.ac.in" class="email">sm3@nibmg.ac.in</a></li>
<li><i class="fa fa-github"></i> Github ID: <a href="https://github.com/samanwoy1989">samanwoy1989</a></li>
<li><i class="fa fa-phone"></i> +91 898-116-3530</li>
<li>For more information, please contact me via email.</li>
<li><a href="https://orcid.org/0000-0002-8660-5670">ORCiD ID</a></li>
<li><i class="fa fa-graduation-cap"></i> <a href="https://scholar.google.com/citations?user=hvi2rgQAAAAJ&amp;hl=en&amp;oi=sra">Google Scholar</a></li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><p>Experienced in Microarray, RNAseq, Single Cell RNAseq data analysis with bioconductor and R</p></li>
<li><p>Skilled in R, LaTeX, Markdown, git</p></li>
<li><p>Python and Bash literate</p></li>
<li><p>Experienced in scientific computing with Linux OS</p></li>
</ul>
</div>
<div id="disclaimer" class="section level2">
<h2>Disclaimer</h2>
<p>This resume was made with the R package <a href="https://github.com/rstudio/pagedown"><strong>pagedown</strong></a>.</p>
<p>Last updated on 2021-04-21.</p>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Samanwoy Mukhopadhyay</h2>
</div>
<div id="education" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education</h2>
<div id="university-of-calcutta" class="section level3">
<h3>University of Calcutta</h3>
<p>B.Sc., Zoology (Honours) with 77.75% marks</p>
<p>2010</p>
</div>
<div id="university-of-calcutta-1" class="section level3">
<h3>University of Calcutta</h3>
<p>M.Sc., Zoology, CGPA-8.56 with grade O (Outstanding)</p>
<p>2012</p>
</div>
<div id="national-institute-of-biomedical-genomics-kalyani" class="section level3">
<h3>National Institute of Biomedical Genomics, Kalyani</h3>
<p>Submitted Ph.D thesis in <em>Host Transcriptome Response in Sepsis</em></p>
<p>2019</p>
</div>
</div>
<div id="research-experience" class="section level2" data-icon="laptop">
<h2>Research Experience</h2>
<ul>
<li>Experienced in transcriptomic data analysis, data mining, handling big data generated from the lab and also from the public domain.</li>
<li>Experienced in performing meta-analysis from public data to draw inference from already available data in the field (GEO, TCGA).</li>
<li>Experienced in creating data/ software packages (in R) for reproducibility of the work done.</li>
<li>All the data and codes are available in public domain for each manuscript as a R vignette.</li>
<li>Proficient in sweave, knitr, Markdown, Latex for scientific document writing.</li>
<li>Experienced in genomic data generation in a laboratory in Gene expression microarray.</li>
<li>Isolation of DNA, RNA from clinical samples, Bioanalyzer (capillary electrophoresis) for quality assessment of the nucleic acids, PCR, gel Electrophoresis, cDNA library preparation for gene expression microarray chips, gene expression microarray.</li>
<li>Literate in basic microbiological experiments.</li>
<li>Experienced lab management. From sample collection to data generation to data analysis and writing of reports.</li>
<li>Experienced to work in collaboration with technical staffs and other lab members to work as a team.</li>
</ul>
<div class="page-break-after"></div>
</div>
<div id="selected-publications" class="section level2" data-icon="file">
<h2>Selected Publications</h2>
<div id="host-response-to-sars-cov-2-insight-from-transcriptomic-studies." class="section level3">
<h3>Host response to SARS-CoV-2: Insight from transcriptomic studies. <a href="http://op.niscair.res.in/index.php/IJBB/article/view/42046"><i class="fa fa-link"></i></a></h3>
<p><strong>Mohapatra SK, Mukhopadhyay S</strong>
<strong>Indian Journal of Biochemistry and Biophysics (IJBB), 58(1), pp.7-12.</strong> (<strong>Review Article</strong>)</p>
<p>2021</p>
</div>
<div id="dynamic-dysregulation-of-il-6-and-genes-functional-in-netosis-complement-and-coagulation-in-severe-covid-19-illness." class="section level3">
<h3>Dynamic dysregulation of IL-6 and genes functional in NETosis, complement and coagulation in severe COVID-19 illness. <a href="https://www.medrxiv.org/content/10.1101/2020.10.13.20211425v1"><i class="fa fa-link"></i></a></h3>
<p><strong>medRxiv</strong>. 2020 Oct 15. (<strong>Pre-print</strong>)<br />
<strong>Mukhopadhyay S</strong>, Sinha S, Mohapatra SK.</p>
<p>2020</p>
</div>
<div id="sepsis-associated-pathways-segregate-cancer-groups." class="section level3">
<h3>Sepsis-associated pathways segregate cancer groups. <a href="https://bmccancer.biomedcentral.com/articles/10.1186/s12885-020-06774-9"><i class="fa fa-link"></i></a></h3>
<p><strong>BMC cancer</strong>. 2020 Apr ;20:1-1.
Tripathi H, <strong>Mukhopadhyay S</strong>, Mohapatra SK.</p>
<p>2020</p>
</div>
<div id="immunosuppression-rather-than-inflammation-is-a-salient-feature-of-sepsis-in-an-indian-cohort." class="section level3">
<h3>Immunosuppression, rather than inflammation, is a salient feature of sepsis in an Indian cohort. <a href="https://www.biorxiv.org/content/10.1101/742924v1"><i class="fa fa-link"></i></a></h3>
<p><strong>bioRxiv</strong>. 2019 Aug 22:742924. (<strong>Pre-print</strong>)
<strong>Mukhopadhyay S</strong>, Thatoi PK, Das BK, Mohapatra SK.</p>
<p>2019</p>
</div>
<div id="transcriptomic-meta-analysis-reveals-up-regulation-of-gene-expression-functional-in-osteoclast-differentiation-in-human-septic-shock." class="section level3">
<h3>Transcriptomic meta-analysis reveals up-regulation of gene expression functional in osteoclast differentiation in human septic shock. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0171689"><i class="fa fa-link"></i></a></h3>
<p><strong>PloS one</strong>. 2017 Feb 15;12(2):e0171689.
<strong>Mukhopadhyay S</strong>, Thatoi PK, Pandey AD, Das BK, Ravindran B, Bhattacharjee S, Mohapatra SK.</p>
<p>2017</p>
</div>
<div id="sepsis-2016-agra-pathway-level-meta-analysis-reveals-transcriptional-signature-of-septic-shock." class="section level3">
<h3>Sepsis 2016 Agra: Pathway-level meta-analysis reveals transcriptional signature of septic shock. <a href="https://ccforum.biomedcentral.com/articles/10.1186/s13054-016-1204-x"><i class="fa fa-link"></i></a></h3>
<p><strong>Critical Care (BMC)</strong>. 2016;20(Suppl 1):5–6. (<strong>Conference Paper</strong>)
<strong>Mukhopadhyay S</strong>, Pandey AD, Bhattacharjee S, Mohapatra SK.</p>
<p>2016</p>
</div>
</div>
<div id="code-and-data-packages" class="section level2" data-icon="laptop">
<h2>Code and Data packages</h2>
<ul>
<li>Published Manuscript: Meta-analysis of Septic Shock: (R data package <a href="https://figshare.com/projects/ssnibmg/18664"><i class="fa fa-link"></i></a>, Vignette<a href="https://figshare.com/projects/ssnibmg/18664"><i class="fa fa-link"></i></a>)</li>
<li>Published Manuscript: Sepsis-associated pathways segregate cancer groups
Rcode: <a href="https://figshare.com/articles/tcnibmgdoc/8118647/5,%20https://figshare.com/articles/tcnibmgdoc/8118647/5"><i class="fa fa-link"></i></a>
Data-package <a href="https://figshare.com/articles/tcnibmg/8118413/3"><i class="fa fa-link"></i></a></li>
<li>Preprint: Host transcriptome analysis for survival of sepsis patients from an Indian cohort: (R data package <a href="https://figshare.com/projects/ssnibmgsurv/67721"><i class="fa fa-link"></i></a>, Vignette
<a href="https://figshare.com/projects/ssnibmgsurv/67721"><i class="fa fa-link"></i></a>)</li>
<li>Preprint: Dynamic dysregulation of IL-6 and genes functional in NETosis, complement and coagulation in severe COVID-19 illness. <a href="https://github.com/samanwoy1989/covid-19"><i class="fa fa-link"></i></a></li>
</ul>
</div>
<div id="honors-and-awards" class="section level2">
<h2>Honors and Awards</h2>
<ul>
<li>Qualified: Joint Council for Scientific &amp; Industrial Research- University Grants Commission (CSIR-UGC) Test for Junior Research Fellowship and Eligibility for Lectureship (NET) held on 17-06-2012. Rank: 65/644 in Life Sciences.</li>
<li>Post-Graduate Merit Scholarship (for university rank holders for PG Programs), University Grants Commission India (U.G.C.); 2010-2012.</li>
<li>Sorashi Bala Choudury Memorial Scholarship (for university rank holders at graduation level), University of Calcutta, 2010.</li>
</ul>
</div>
<div id="teaching-experience" class="section level2" data-icon="chalkboard-teacher">
<h2>Teaching Experience</h2>
<ul>
<li>Participated as an instructor in “Introduction to R and analysing NGS data with R”. Winter school on Analysing Genomic data, NIBMG 2017.</li>
<li>Lab tour instructor in DNB workshop for population genetics, Feb. 2017, NIBMG.</li>
<li>Participated as an instructor in “Introduction to R”, NIBMG summer school for systems biology, July 2018.</li>
<li>Participated as an instructor in NIBMG Clinical Genetics workshop, Dec,2019.</li>
</ul>
</div>
<div id="references" class="section level2">
<h2>References</h2>
<table>
<colgroup>
<col width="33%" />
<col width="33%" />
<col width="33%" />
</colgroup>
<tbody>
<tr class="odd">
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><strong>Dr.Saroj Kant Mohapatra</strong><br>Assistant Professor<br>National Institute of<br>Biomedical Genomics<br>Kalyani 741251, India<br>email: <a href="mailto:skm1@nibmg.ac.in" class="email">skm1@nibmg.ac.in</a></td>
<td></td>
<td><strong>Dr.Samsiddhi Bhattacharjee</strong><br>Associate Professor<br>National Institute of<br>Biomedical Genomics<br>Kalyani 741251, India<br>email:<a href="mailto:sb1@nibmg.ac.in" class="email">sb1@nibmg.ac.in</a></td>
</tr>
<tr class="odd">
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><strong>Dr.Analabha Basu</strong><br>Associate Professor<br>National Institute of<br>Biomedical Genomics<br>Kalyani 741251, India<br>email:<a href="mailto:ab1@nibmg.ac.in" class="email">ab1@nibmg.ac.in</a></td>
<td></td>
<td><strong>Prof. Partha Pratim Majumder</strong><br>Distinguished Professor<br>National Institute of<br>Biomedical Genomics<br>Kalyani 741251, India<br>email:<a href="mailto:ppm1@nibmg.ac.in" class="email">ppm1@nibmg.ac.in</a></td>
</tr>
</tbody>
</table>
<!-- Dr.Saroj Kant Mohapatra -->
<!-- Assistant Professor -->
<!-- National Institute of -->
<!-- Biomedical Genomics -->
<!-- Kalyani 741251, India -->
<!-- email: skm1@nibmg.ac.in -->
<!-- Dr.Samsiddhi Bhattacharjee -->
<!-- Associate Professor -->
<!-- National Institute of -->
<!-- Biomedical Genomics -->
<!-- Kalyani 741251, India -->
<!-- email:sb1@nibmg.ac.in -->
<!-- Dr.Analabha Basu -->
<!-- Associate Professor -->
<!-- National Institute of -->
<!-- Biomedical Genomics -->
<!-- Kalyani 741251, India -->
<!-- email:ab1@nibmg.ac.in   -->
<!-- Prof. Partha Pratim Majumder -->
<!-- Distinguished Professor -->
<!-- National Institute of -->
<!-- Biomedical Genomics -->
<!-- Kalyani 741251, India -->
<!-- email:ppm1@nibmg.ac.in -->
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    // if there is no icon, add an icon:
    if (el.firstElementChild.firstChild && el.firstElementChild.firstChild.nodeName !== 'I') {
      s = el.dataset['icon'] || 'bookmark';
      el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
    }
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
